content/reference-rippled.md

```diff
@@ -10693,6 +10701,311 @@ The response follows the [standard format](#response-formatting), with a success
+            "n9Ltz6ZxPRWTkqwBbpvgbaXPgm6GYCxCJRqFgNXhWVUebgezo28H"
+        ],
+        "validation_quorum":2,
+        "validator_list_expires":"2017-Oct-13 14:56:00"
+    },
+    "status":"success"
+}
+```
+
+*Commandline*
+
+```
+Loading: "/etc/rippled.cfg"
+Connecting to 127.0.0.1:5005
+{
+    "result":{
+        "local_static_keys": [],
+        "publisher_lists":[
+            {
+                "available":true,
+                "expiration":"2017-Oct-13 14:56:00",
+                "list":[
+                    "n9Ltz6ZxPRWTkqwBbpvgbaXPgm6GYCxCJRqFgNXhWVUebgezo28H",
+                    "n94D73ZKSUaTDCnUqYW5ugJ9fHPNxda9GQVoWA6BGtcKuuhozrD1"
+                ],
+                "pubkey_publisher":"ED58ED4AA543B524F16771F6E1367BAA220D99DCF22CD8CF7A11309E9EAB1B647B",
+                "seq":1,
+                "version":1
+            }
+        ],
+        "status":"success",
+        "trusted_validator_keys":[
+            "n94D73ZKSUaTDCnUqYW5ugJ9fHPNxda9GQVoWA6BGtcKuuhozrD1",
+            "n9Ltz6ZxPRWTkqwBbpvgbaXPgm6GYCxCJRqFgNXhWVUebgezo28H"
+        ],
+        "validation_quorum":2,
+        "validator_list_expires":"2017-Oct-13 14:56:00"
+    },
+    "status":"success"
+}
+```
+
+<!-- MULTICODE_BLOCK_END -->
+
+The response follows the [standard format](#response-formatting), with a successful result containing the following fields:
+
+| `Field`                  | Type   | Description                              |
+|:-------------------------|:-------|:-----------------------------------------|
+| `listed_static_keys`     | Array  | Array of public keys for validators always eligible for inclusion in the trusted list. |
+| `publisher_lists`        | Array  | Array of publisher list objects.         |
+| `trusted_validator_keys` | Array  | Array of public keys for currently trusted validators. |
+| `validation_quorum`      | Number | Minimum number of trusted validations required to validate a ledger version. Some circumstances may cause the server to require more validations. |
+| `validator_list_expires` | String | Either the human readable time when the current validator list will expire or the string `unknown` if the server has yet to load a published validator list. |
+
+Each member of the `publisher_lists` array is an object with the following fields:
+
+| `Field`            | Type             | Description                              |
+|:-------------------|:-----------------|:-----------------------------------------|
+| `available`        | Boolean          | If `false`, the validator keys in `list` may no longer be supported by this publisher. |
+| `expiration`       | String           |  Either the human readable time when this published list will expire or the string `unknown` if the server has yet to load a list for this publisher.      |
+| `list`             | Array            | Array of published validator keys.   |
+| `pubkey_publisher` | String           | Hex encoded public key of publisher. |
```

Comment:
What type of key must publishers use? ECDSA? Are there other options?

Reply:
It supports whatever the `PublicKey` class supports, which is currently secp25k1 or ed25519, but I think the external tooling for generating keys does ed25519 by default. What is the best designation for that?

Reply:
I'd just change the description to something like this:

> Ed25519 or ECDSA public key of the list publisher, as hexadecimal.